src/pages/starter-kit/reference-app.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Reference applications
+description: Review reference applications created using the Adobe Commerce integration starter kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Reference Applications
+
+This page provides developers with a basic understanding of the types of applications they can create with Adobe Commerce integration starter kit.
+
+## Microsoft Dynamics Finance and Operations Accelerator for Adobe Commerce
+
+Microsoft's Dynamics 365 Finance and Operations (D365 F&O) is an enterprise resource planning (ERP) tool for medium and large businesses to handle their operational processes, such as finance, sales, purchase orders, and more.
+
+This reference application provides a base implementation for the most common data flows you can use when integrating Adobe Commerce with D365 F&O.
+
+### Features
+
+- **Out-of-the-Box Integration** - Leverage Adobe’s serverless extensibility architecture to integrate data between Adobe Commerce and Microsoft Dynamics Finance & Operations out-of-the-box.
```

Comment:
Consider using the D365 F&O abbreviation instead of spelling out name each time.